### ISSUE TITLE
Resources Relating to TOR, I2P, and VPN use with Bitcoin

### ISSUE THREAD

**Issue by ABISprotocol** [btcfoundationedcom/btcfoundationedcom.github.io#32]
This issue is opened to help gather resources (primarily if you know of good blog posts, instructionals, etc.) that can be provided here that indicate TOR / TOR relay / OnionTip / I2P / VPN use with Bitcoin.  Ultimately the purpose of this will be first, just to gather these resources (hopefully you have some confidence in them, having tried them) and put a link to them below.  But then the resources will be used to provide a kind of larger guide that will help users through the process of setting up TOR with bitcoin (or exploring using VPN with bitcoin or I2P, etc.) no matter what operating system they are using.  Thanks in advance for any contributions you can provide below.

**Comment by ABISprotocol** [comment]
Example links:  [Configuring a Tor relay on Debian/Ubuntu](https://www.torproject.org/docs/tor-relay-debian.html.en)
[OnionTip](https://oniontip.com/)

**Comment by sressler** [comment]
Would resources dealing with coin mixing belong here also? or perhaps that's another topic.

**Comment by ABISprotocol** [comment]
Yes, you can add resources for coin mixing here. However, the guide we want to produce will primarily focus on TOR with bitcoin (and I2P or VPN with bitcoin, etc) no matter what operating system you are using.  Bitcoin mixers are up to the reader of the guide, we can mention that they exist and perhaps list some examples and also some words of caution (if people are going to use TOR with bitcoin and also use a bitcoin mixing service, or use bitcoin without tor and use a bitcoin mixing service, they should be aware of the risks of using such a service).  The primary goal of this as I see it is to ensure that some kind of privacy protecting service is "always on" for people who are reading the guide that we end up creating and that it is dead dumb simple (or at least has relatively simplistic step by step instructions that nearly anyone could walk through without stumbling to ensure they get TOR working with bitcoin on their machine).

**Comment by ABISprotocol** [comment]
@sressler More of an answer to your question is now available being as there has been some coin mixing (specifically, CoinJoin) implementation that has been realized.  
See http://joinmarket.io/ | https://github.com/JoinMarket-Org/joinmarket/blob/master/README.md
and https://github.com/AdamISZ/JMBinary/blob/master/README.md

I don't yet have any strong recommendations for VPN or I2P guidance (for how-to use it a certain way with bitcoin) but with any resources or guidance please post.
